retrieval/helpers_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	"time"
 
 	clientmodel "github.com/prometheus/client_golang/model"
+	"github.com/prometheus/prometheus/config"
```

Comment:
Space above this line (separate import group for repo-internal imports - yeah, bit weird rule, might get rid of it at some point).